types_int.go

```diff
@@ -1,8 +1,12 @@
 package wypes
 
+import "encoding/binary"
+
 // Int8 wraps [int8], a signed 8-bit integer.
 type Int8 int8
 
+const Int8Size = 1
```

Comment:
Used only internally, should be private

```suggestion
const int8Size = 1
```

Reply:
Corrected.